saleor/static/dashboard-next/home/fixtures.ts

```diff
@@ -0,0 +1,101 @@
+export const shop = (placeholderImage: string) => ({
```

Comment:
This fixture should be defined as `const shop: (placeholderImage: string) => DashboardProps = placeholderImage => ({})`